code/modules/mining/drilling/drill.dm

```diff
@@ -234,6 +245,7 @@
 
 	if(error)
 		src.visible_message("<span class='notice'>\The [src] flashes a '[error]' warning.</span>")
+		faultreporter.autosay(error, src.name, "Supply")
```

Comment:
Could you give an example of the message this gives?

Reply:
![image](https://user-images.githubusercontent.com/14942479/36862077-b7884ca8-1d7d-11e8-99c2-e626f40d415b.png)

Reply:
that looks super off but i dont have solutions so i guess i wont complain, its an upgrade either way

Reply:
Fix the error messages to be properly capitalized?

Reply:
Done.